README.md

```diff
@@ -1 +1,71 @@
-"# Internet-ShopRepository" 
+<h1>"Internet-Shop Project"</h1>
+<br><hr>
+
+# Table of Contents
+* [Description](#Description)
+* [Project structure](#Structure)
+* [Start](#Start)
+* [For user](#User)
+* [For administrator](#Admin)
+* [For developer](#Developer)
+* [Author](#Author)
+<hr>
+
+# <a name="Description"></a>Description
+It's simple realization of the web-application - internet shop.
+This shop has simple UI which done by CSS and JSP pages(HTML, JSPTL).
+Project has: authentic and authorization filter.
+Dao layer with tro implementation: Lists and JDBC.
+Service layer with logic between Dao and Servlets.
+View layer with pages.
+<hr>
+
+# <a name="Structure"></a>Project Structure
+* Java 11
+* Maven 4.0.0
+* Tomcat 8.5.50
+* Servlet 3.1
+* MySQL 8.0.17
+* Logger log4j 1.2.17
+* JSTL 1.2
+<hr>
+<br>
+
+# <a name="Start"></a>Start
+* Click "Inject" button. Or you can register two users and to do the next steps.
+* Change admin role_id in table user_roles from 1 to 2.
+* And after you can login as user with login - user and password - user.
+* And as admin with login - admin and password - admin.
+<hr>
+<br>
+
+# <a name="User"></a>For user
+* User can watch item list</h5>
```

Comment:
User can:
- watch item list
- add and delete items into his bucket
- create order from bucket with items
- delete your orders